caffe2/python/operator_test/instance_norm_test.py

```diff
@@ -76,7 +76,16 @@ def test_instance_norm_gradients(
             store_inv_stdev=store_inv_stdev,
             epsilon=epsilon,
             order=order)
-        input_blobs = self._get_inputs(N, C, H, W, order)
+
+        input_data = np.arange(N * C * H * W).astype(np.float32)
```

Comment:
why moved away from `_get_inputs`?

Reply:
_get_inputs used rand to generate input, whose range is pretty small and may lead to numerical failures for grad check. Here we use arrange and shuffle to generate a tensor those elements are far away from each other.

Reply:
I see, shall we just extend _get_inputs a bit? Since most of the code looks very similar

Reply:
The correct thing is just remove _get_inputs at all. There is no need for 3 lines in such a function.